FieldOpt/Runner/runners/synchronous_mpi_runner.cpp

```diff
@@ -215,7 +215,7 @@ void SynchronousMPIRunner::Execute() {
                 if (simulation_success) {
                     tag = MPIRunner::MsgTag::CASE_EVAL_SUCCESS;
                     printMessage("Setting objective function value.", 2);
-                    worker_->GetCurrentCase()->set_objective_function_value(objective_function_->value());
+                    worker_->GetCurrentCase()->set_objective_function_value(objective_function_->value(model_->wellCost(settings_->optimizer()->objective())));
```

Comment:
I'm not happy with how value has to be called here. We should discuss better alternatives (nothing comes to mind right now, but I'll think about it)